### DISCUSSION TITLE
Whisper suddenly gave an error - ImportError: /usr/lib/libtorch_cpu.so: undefined symbol: cblas_gemm_f16f16f32

### DISCUSSION THREAD

**Discussion by flavio92ux** [openai/whisper#2157] [Q&A] [answered]
Hi.

I installed Whisper in ArchLinux distro, at first runs. But, after some days, doesnt work. Error message:

Traceback (most recent call last):
  File "/usr/bin/whisper", line 5, in <module>
    from whisper.transcribe import cli
  File "/usr/lib/python3.12/site-packages/whisper/__init__.py", line 8, in <module>
    import torch
  File "/usr/lib/python3.12/site-packages/torch/__init__.py", line 237, in <module>
    from torch._C import *  # noqa: F403
    ^^^^^^^^^^^^^^^^^^^^^^
ImportError: /usr/lib/libtorch_cpu.so: undefined symbol: cblas_gemm_f16f16f32

What happen?

**Comment by Levitiku5** [comment, marked as answer]
Check out the developer response here: https://www.reddit.com/r/archlinux/comments/1ciq91p/cannot_use_pytorch_cblas_symbol_undefined_in/

This is an issue with the Python module PyTorch, and according to the response it will be corrected shortly.

**Reply by flavio92ux** [reply]
Thanks
